packages/mock-app/src/pages/Verify.tsx

```diff
@@ -4,7 +4,7 @@ import { IVerifyResult, VerifiableCredential } from '@vckit/core-types';
 import { useLocation } from 'react-router-dom';
 import { computeEntryHash } from '@veramo/utils';
 import { Status } from '@mock-app/components';
-import { publicAPI, privateAPI } from '@mock-app/services';
+import { publicAPI, privateAPI, verifyVC } from '@mock-app/services';
```

Comment:
It seems the `verifyVC` function already wraps the API call to verify the VC, so I don’t think we need to import the `privateAPI` function here.

Reply:
Removed